src/pages/vedtak/Vedtaksoppsummering.tsx

```diff
@@ -1,137 +1,55 @@
+import Revurderingoppsummering from '~features/revurdering/revurderingoppsummering/Revurderingoppsummering';
 import { useI18n } from '~lib/hooks';
 import * as Routes from '~lib/routes';
-import { IverksattRevurdering } from '~types/Revurdering';
 import { Sak } from '~types/Sak';
-import { VedtakType } from '~types/Vedtak';
 
+import { hentInformasjonKnyttetTilVedtak } from './utils';
 import messages from './vedtaksoppsummering-nb';
 import styles from './vedtaksoppsummering.module.less';
 
 interface Props {
     sak: Sak;
 }
 
-const vedtaksresultatToTekst = (type: VedtakType, intl: IntlShape): string => {
-    switch (type) {
-        case VedtakType.SØKNAD:
-            return intl.formatMessage({ id: 'vedtaktype.søknad' });
-        case VedtakType.AVSLAG:
-            return intl.formatMessage({ id: 'vedtaktype.avslått' });
-        case VedtakType.ENDRING:
-            return intl.formatMessage({ id: 'vedtaktype.endring' });
-        case VedtakType.OPPHØR:
-            return intl.formatMessage({ id: 'vedtaktype.opphør' });
-        case VedtakType.INGEN_ENDRING:
-            return intl.formatMessage({ id: 'vedtaktype.ingenendring' });
-        case VedtakType.REGULER_GRUNNBELØP:
-            return intl.formatMessage({ id: 'vedtaktype.regulergrunnbeløp' });
-    }
-};
-
-/* TODO ai 16.03.2021: Denna støtter pt innvilgede revurderingsvedtak. Må sørge for att søknadsbehandling og andre resultat støttes i framtiden. */
 const Vedtaksoppsummering = (props: Props) => {
     const urlParams = Routes.useRouteParams<typeof Routes.vedtaksoppsummering>();
     const intl = useI18n({ messages });
-    const [fetchVedtaksbrev, setFetchVedtaksbrev] = useState<RemoteData.RemoteData<ApiError, null>>(RemoteData.initial);
+    const history = useHistory();
     const vedtak = props.sak.vedtak.find((v) => v.id === urlParams.vedtakId);
     if (!vedtak) return <div>{intl.formatMessage({ id: 'feilmelding.fantIkkeVedtak' })}</div>;
 
-    const revurderingSomFørteTilVedtak = props.sak.revurderinger.find(
-        (b) => b.id === vedtak.behandlingId
-    ) as IverksattRevurdering;
-
-    const hentVedtaksbrev = async () => {
-        setFetchVedtaksbrev(RemoteData.pending);
-
-        const res = await fetchBrevutkastForRevurdering(props.sak.id, vedtak.behandlingId);
-        if (res.status === 'ok') {
-            setFetchVedtaksbrev(RemoteData.success(null));
-            window.open(URL.createObjectURL(res.data));
-        } else {
-            setFetchVedtaksbrev(RemoteData.failure(res.error));
+    const vedtaksinformasjon = hentInformasjonKnyttetTilVedtak(props.sak, vedtak);
+
+    const renderOppsummering = () => {
+        switch (vedtaksinformasjon?.type) {
+            case 'revurdering':
+                return (
+                    <Revurderingoppsummering
+                        revurdering={vedtaksinformasjon.revurdering}
+                        grunnlagsdataOgVilkårsvurderinger={
```

Comment:
Kunne denne het `forrigeGrunnlagsdataOgVilkårsvurderinger`? Hvordan vil vi egentlig vise denne? Si typisk forrige vedtak var 12 måneder. Og denne revurderinga er 1 måned. Da vil vi sammenligne den ene måneden mot 12 måneder :shrug:

Reply:
Denne bruks ju allerede for å oppsummere revurderinger? Viser bara relevant information innenfor revurderingsperioden.

F. eks:
![image](https://user-images.githubusercontent.com/26689337/123920244-081e9880-d986-11eb-89ea-368e34a93530.png)